src/db.cpp

```diff
@@ -84,17 +84,28 @@ void DbTable::checkTransaction()
 {
     if (mCheckAlwaysTransacted)
     {
-        assert(mCurrentTransactionCommiter);  // if this fails, we should have started a DBTableTransactionCommitter higher in the call stack
-        if (mCurrentTransactionCommiter)
+        assert(mTransactionCommitter);  // if this fails, we should have started a DBTableTransactionCommitter higher in the call stack
+        if (mTransactionCommitter)
         {
-            mCurrentTransactionCommiter->beginOnce();
+            mTransactionCommitter->beginOnce();
         }
     }
 }
 
+void DbTable::resetCommitter()
+{
+    if (mTransactionCommitter)
+    {
+        mTransactionCommitter->commitNow();
```

Comment:
We may not want to commit the open transaction in all cases. Remember the `sctable` should only commit upon reception of a new `scsn` via actionpackets (or a bit later when the ongoing cs requests complete).
Please, check that it's not possible to do a commit by mistake, or the account's state may get corrupted. Thanks

Reply:
Yeah, I think you're right. In fact, I think we shouldn't commit here at all as this function is only called from the table destructor. Anything that wasn't committed until then should be discarded (I assume that's the reason why we're calling `abort` in the table destructor?!).

Reply:
Ok. Done. And now the method name makes more sense :)